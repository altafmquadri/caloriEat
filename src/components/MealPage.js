import React from 'react';
import MealSummary from './MealSummary';
import MealList from './MealList';

const MealPage = () => {
    return ( 
        <div>
            <MealSummary />
            <MealList />
        </div>
     );
}
 
export default MealPage;