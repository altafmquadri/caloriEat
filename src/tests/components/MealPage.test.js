import React from 'react';
import {shallow } from 'enzyme'
import MealPage from '../../components/MealPage'

describe('<MealPage />', () => {
    test('should render the MealPage component', () => {
        const wrapper = shallow(<MealPage/>)
        expect(wrapper).toMatchSnapshot()
    })
})