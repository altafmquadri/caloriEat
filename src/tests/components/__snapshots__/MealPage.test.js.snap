// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MealPage /> should render the MealPage component 1`] = `
<div>
  <Connect(MealSummary) />
</div>
`;
